mep/books/models.py

```diff
@@ -30,38 +37,127 @@ class PublisherPlace(Named, Notable):
 
 class Publisher(Named, Notable):
     '''Model for publishers'''
-    pass
+
+
+class Subject(models.Model):
+    '''Linked data subjects for describing :class:`Item`'''
+
+    #: name/label for the subject (required but not required unique)
+    name = models.CharField(max_length=255)
+    #: linked data URI for the subject
+    uri = models.URLField("URI", help_text="Subject URI", unique=True)
+    #: rdf type for the subject
+    rdf_type = models.URLField("RDF Type")
+
+    def __str__(self):
+        return '%s (%s)' % (self.name, self.uri)
+
+    def __repr__(self):
+        return '<Subject %s (%s)>' % (self.uri, self.name)
+
+    @classmethod
+    def create_from_uri(cls, uri: str) -> 'Subject':
+        '''Initialize a new :class:`Subject` from a URI. Loads the URI
+        as an :class:`rdflib.Graph` in order to pull the preferred label
+        and RDF type for the URI.'''
```

Comment:
Might be worth noting the explicit `raise_from_status` in case we ever use this elsewhere to make it less likely to wrap it in `try`/`except` using `:raises` notation.

Reply:
I think I left it because I was ambivalent about that solution for error handling.  It worked fine until I had to add a check for html responses, which meant there were two different error conditions (exception or nothing returned). Looking at it now, I think it would be better to drop the exception `raise_for_status` call and just add the error logging here instead of in the calling method.